data/data-pipeline/data_pipeline/etl/score/etl_score.py

```diff
@@ -56,8 +56,6 @@ def __init__(self):
         self.ISLAND_DEMOGRAPHIC_BACKFILL_FIELDS: List[str] = []
 
     def extract(self) -> None:
-        logger.info("Loading data sets from disk.")
```

Comment:
For a second I was like "wait but this is true," but then I remembered you'd instead put these centrally as debug in `_run_one_dataset`, which makes a ton more sense. This is a not for a future reviewer who also goes "wait!?"

Reply:
Yep, I tried to delete these if they didn't add value.